cogs/info.py

```diff
@@ -286,7 +286,7 @@ async def on_message(self, payload):
         output_msg = ""
 
         # ignore user commands, as well as responses by the bot
-        if(msg_str[0] == "!" or payload.author.bot):
+        if(payload.author.bot or msg_str[0] == "!" ):
```

Comment:
Is there a functional reason for this change?

Reply:
Because embeds technically have no content, when a message with just an embed was read, the bot freaked out. Switching this around is a half solution as long as no other bots use embeds. A better solution might be to add another check to see if their embeds, since only bots can produce those.

Reply:
This is more than good enough for now. All good on my front, thank you!!